WordPress/WordPressUITests/Screens/TabNavComponent.swift

```diff
@@ -15,12 +14,14 @@ class TabNavComponent: BaseScreen {
         readerTabButton = tabBars.buttons["readerTabButton"]
         writeTabButton = XCUIApplication().buttons["floatingCreateButton"]
         notificationsTabButton = tabBars.buttons["notificationsTabButton"]
-        meTabButton = tabBars.buttons["meTabButton"]
-        super.init(element: meTabButton)
+        super.init(element: mySitesTabButton)
     }
 
     func gotoMeScreen() -> MeTabScreen {
-        meTabButton.tap()
+        gotoMySitesScreen()
+        app.cells[WPUITestCredentials.testWPcomSitePrimaryAddress].tap()
+        let meButton = app.navigationBars.buttons["meBarButton"]
+        meButton.tap()
         return MeTabScreen()
     }
```

Comment:
We should move the method that opens the Me screen out of `TabNavComponent` (which represents the bottom navigation bar in the app) and into `MySiteScreen` (since the button to open `Me` is now an element only on that screen). That way, our page objects only include elements and actions that are on that screen (or, in the case of the bottom nav bar, in that component) and the movement through different screens the app is more transparent in the tests.

That said, as long as the tests are passing like this, I'm ok with that change being done in a followup PR.

Reply:
This makes sense, and I would do another PR (even right after this) so it's specifically focused on UI Test refactoring.

Reply:
Perfect, thank you!